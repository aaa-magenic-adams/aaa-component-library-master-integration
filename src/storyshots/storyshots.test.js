import initStoryshots from '@storybook/addon-storyshots';
import { mount } from 'enzyme';

initStoryshots({
  renderer: mount,
});
