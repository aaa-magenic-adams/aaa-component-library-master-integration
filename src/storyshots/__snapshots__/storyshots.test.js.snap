// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atomic|Button all buttons 1`] = `ReactWrapper {}`;

exports[`Storyshots Atomic|Button dynamic button 1`] = `ReactWrapper {}`;

exports[`Storyshots Atomic|Input input 1`] = `ReactWrapper {}`;

exports[`Storyshots Atomic|Input states 1`] = `ReactWrapper {}`;

exports[`Storyshots Atomic|Link text link 1`] = `ReactWrapper {}`;

exports[`Storyshots Atomic|SelectList Usage and States 1`] = `ReactWrapper {}`;

exports[`Storyshots Atomic|Typography Text sizes 1`] = `ReactWrapper {}`;

exports[`Storyshots Molecules|ButtonGroup primary and secondary 1`] = `ReactWrapper {}`;

exports[`Storyshots Molecules|Form basic form + code demo 1`] = `ReactWrapper {}`;

exports[`Storyshots Molecules|Form basic form 1`] = `ReactWrapper {}`;

exports[`Storyshots Molecules|Stepper numerical  1`] = `ReactWrapper {}`;

exports[`Storyshots Molecules|ToggleButtonGroup Usage and States 1`] = `ReactWrapper {}`;

exports[`Storyshots Welcome|Storybook Welcome description 1`] = `ReactWrapper {}`;
